### ISSUE TITLE
Disable Crippling Shrines isn't generated being generated in the config

### ISSUE THREAD

**Issue by tyreforhyred** [diasurgical/devilutionX#2604]
**Important information**
Operating system / DevilutionX version / did you download a release or did you compile yourself?
windows 10/ DevilutionX 1.2.1/ release

**Describe**
Disable Crippling Shrines isn't generated along with the other config options in the diablo.ini file

**To Reproduce**
delete diablo.ini and start up devilution, exit the game and open the config

**Expected behavior**
for Disable Cripple Shrines to be generated along the rest

**Comment by ThomasChr** [comment]
This one also does seem to be simple. I would like to try if no one else wants.

**Comment by ThomasChr** [comment]
Can't reproduce on Linux (latest commit 54b60101f55c8d32e6c6f0fae74cdc10d5e3f4ef), 'Disable Crippling Shrines' has been generated:
![image](https://user-images.githubusercontent.com/13014163/129486485-2a34e798-f44b-4529-a6dd-9002cf800830.png)

After digging a little bit deeper I found commit bf908da5bcb82bddee4f15cbc1103a29824a3f1b which added the flag. This commit was on 2021-04-19 - Version 1.2.1 was released on 2021-04-13.

So it won't be generated because it wasn't there then.
